app/src/main/res/xml/preferences_gpodder.xml

```diff
@@ -26,10 +26,10 @@
     <Preference
             android:key="pref_gpodnet_hostname"
             android:title="@string/pref_gpodnet_sethostname_title"/>
-    <SwitchPreferenceCompat
+<!--    <SwitchPreferenceCompat
```

Comment:
Please delete instead of commenting. The setting is no longer used in the old location, so the code does not have to be kept either.

Reply:
removed all the commented code.